StreamingAnalytics/LowLatencySample/Temperature-Epl/monitors/testing.mon

```diff
@@ -0,0 +1,27 @@
+monitor Testing {
+	
+	
+	event Trigger {}
+	action onload() {
+		spawn testCode();
+	}
+	
+	float counter := 100.0;
+	
+	action testCode() {
+		monitor.subscribe("out");
+
+		
+		on all Trigger() {
+			float reading := counter.rand();
+			log "Sending "+reading.toString() ;
```

Comment:
I'd recommend putting a space around the '+' operator, I think it's better for readability.